src/main/java/seedu/address/model/Cookbook.java

```diff
@@ -117,4 +117,8 @@ public boolean equals(Object other) {
     public int hashCode() {
         return recipes.hashCode();
     }
+
+    public boolean contains(Recipe recipe) {
+        return recipes.contains(recipe);
+    }
```

Comment:
For both `Cookbook` and `Inventory`, there's already `hasRecipe()` and `hasIngredient()` which do the same thing. Instead of adding the `contains()` methods to both, maybe we could get CookingPapa/AddressBook to use the `hasRecipe()` and `hasIngredient()` methods instead of the `contains()` method.

Reply:
right, fixed